app/frontend/src/router/index.js

```diff
@@ -1,10 +1,10 @@
 import Vue from 'vue';
 import VueRouter from 'vue-router';
 
-import CDOGS from '../views/CDOGS.vue';
-import Dashboard from '../views/Dashboard.vue';
-import Home from '../views/Home.vue';
-import NotFound from '../views/NotFound.vue';
+const CDOGS = () => import('../views/CDOGS.vue');
```

Comment:
are there other places in the code to do this? just to make it consistent, and if it's a good pattern it'll get used again.

Reply:
As far as I can tell this only affects vue router stuff. This pattern wouldn't be applicable on other places such as component imports as far as I can tell.
https://router.vuejs.org/guide/advanced/lazy-loading.html